### PR TITLE
[mod_sofia] Parse extra headers on reINVITE

### PR DESCRIPTION
**Pull request by davidcsi**

### CODE REVIEW

**Comment by davidcsi** [comment]
result

```
2022-11-30 21:37:27.550128 94.80% [ERR] mod_commands.c:7206 Event Type: CUSTOM
2022-11-30 21:37:27.550128 94.80% [ERR] mod_commands.c:7206 action: Answer
```

**Comment by hibrid** [comment]
Is this something that can be ran by signalwire to see if they have concerns?

**Review by hibrid — DISMISSED** [review]

**Review by anthmFS — APPROVED** [review]

File: src/mod/endpoints/mod_sofia/sofia.c [hunk 1]
```diff
@@ -10308,7 +10308,7 @@ void sofia_handle_sip_i_reinvite(switch_core_session_t *session,
 				tech_pvt->mparams.last_sdp_str = tech_pvt->mparams.prev_sdp_str;
 			}
 		}
-
+		sofia_glue_set_extra_headers(session, sip, SOFIA_SIP_HEADER_PREFIX);
```
Comment (andywolk) [inline src/mod/endpoints/mod_sofia/sofia.c:10311]: Looks good to me except that we need a new line after an if here.

**Review by andywolk — CHANGES_REQUESTED** [review]

File: src/mod/endpoints/mod_sofia/sofia.c (hunk 1 above)
Comment (davidcsi) [reply src/mod/endpoints/mod_sofia/sofia.c:10312]: @andywolk I added an extra blank line after the closing bracket, is that what you mean? not really sure.

**Review by andywolk — APPROVED** [review]

**Pull request merged**